src/main/resources/application.properties

```diff
@@ -1,16 +1,23 @@
 server.port: 9001
 server.contextPath:
 security.basic.enabled: false
+
+# Bean overriding is necessary for TokenAuthorizationFilter.
+spring.main.allow-bean-definition-overriding: true
```

Comment:
For these required overrides, it would be best we have an issue created to identify the problem.

Reply:
Will create an issue for the same